scripts/prisma_version_stable

```diff
@@ -0,0 +1 @@
+x
```

Comment:
Shouldn't there be the version number instead of `x`?

Reply:
The PR starts with x and the first commit replaces this with the actual version number in the CI (and releasing in the process)